### PR TITLE
deprecate repository

### PR DESCRIPTION
**Pull request by ktdreyer**
Remove all the files in the repository and point users at the new `Containerfile` location.

### CODE REVIEW

**Review by guits — APPROVED** [review]

**Comment by guits** [comment]
@ktdreyer I wonder if removing everything might be a bit too "aggressive" for now. Shouldn't we just update the `README.md` to inform people that the repository is deprecated and redirect them to the `Containerfile` location?

**Comment by ktdreyer** [comment]
The files will still be in the Git history. Removing the files makes it clear to users and contributors that developers are focused on the new area.

**Comment by travisn** [comment]
Why not just archive the repo so it's readonly? And how will they know where to find new development if the readme is even empty?

**Comment by guits** [comment]
> And how will they know where to find new development if the readme is even empty?

Correct. Although I still think that removing everything is a bit aggressive, I would at least leave a comment in README.md to let users know what's happening.

**Comment by ktdreyer** [comment]
Whoops, sorry! I wrote a new README but I forgot to push it in this PR. I've amended and re-pushed. It can be a little difficult to read in this large GitHub diff, so I'll paste the README contents here:

--------

`Containerfile` has moved
-------------------------

This `ceph-container` GitHub repository is deprecated and read-only. **The Ceph developers have moved all development to the `Containerfile` in the primary `ceph` GitHub repository:**

* `main`: https://github.com/ceph/ceph/blob/main/container/Containerfile

Older releases:

* `squid`: https://github.com/ceph/ceph/blob/squid/container/Containerfile
* `reef`: https://github.com/ceph/ceph/blob/squid/container/Containerfile
* `quincy`: https://github.com/ceph/ceph/blob/quincy/container/Containerfile

Please make future contributions to these locations.


History
-------

This repository originally held the first experiments of putting Ceph into container images. Later on, this repository held a variety of operating systems and versions in file snippets with a templating system. We have simplified this to a single flat `Containerfile`, one per Ceph release.

**Review by guits — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,160 +1,20 @@
...
+* `main`: https://github.com/ceph/ceph/blob/main/container/Containerfile
 
-Build Ceph into container images with upstream support for the latest few Ceph
-releases on CentOS. ceph-container also supports builds for multiple distributions.
+Older releases:
 
-Find available container image tags
------------------------------------
+* `squid`: https://github.com/ceph/ceph/blob/squid/container/Containerfile
+* `reef`: https://github.com/ceph/ceph/blob/squid/container/Containerfile
```
Comment (guits) [inline README.md]:
```suggestion
* `reef`: https://github.com/ceph/ceph/blob/reef/container/Containerfile
```

**Review by guits — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ktdreyer) [reply README.md]: Thank you for catching this. I've amended and re-pushed.

**Review by guits — APPROVED** [review]

**Comment by ktdreyer** [comment]
@dmick , is this ready to merge now?

**Comment by dmick** [comment]
We should disable/delete the Jenkins jobs that use it first, probably.
Working on that and the change to the third release "ceph" job to add the
new container build

**Comment by ktdreyer** [comment]
Now that https://github.com/ceph/ceph-build/pull/2305 is done, @dmick is this read to merge?

**Comment by dmick** [comment]
I think so.  I think https://github.com/ceph/ceph/pull/60936 is too if I could get a review 😄

**Review by dmick — APPROVED** [review]

**Pull request merged**